plugins/providers/metabase/provider.go

```diff
@@ -62,10 +62,21 @@ func (p *provider) GetResources(pc *domain.ProviderConfig) ([]*domain.Resource,
 			db.ProviderType = pc.Type
 			db.ProviderURN = pc.URN
 			resources = append(resources, db)
+
+			if _, ok := resourceTypes[ResourceTypeTable]; ok && resourceTypes[ResourceTypeTable] {
```

Comment:
same here also.

Reply:
I am checking this to confirm the value in the map is true.

Reply:
In what cases, would the key exist and the value be `false` ?